ssqatest/tests/backend/misc_tests/test_create_new_product.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from ssqatest.src.api_helpers.ProductsAPIHelper import ProductsAPIHelper
+from ssqatest.src.dao.products_dao import ProductsDAO
+from ssqatest.src.utilities.genericUtilities import generate_random_string
+import logging as log
+pytestmark= [pytest.mark.backend]
+
+"""
+    This is a test to create a product using API post call and then asserting it was created.
+"""
+
+@pytest.mark.be001
+#create a payload and run a post API call to create a product
```

Comment:
Add this comment inside the function or make it a doc string. It is kinda weird to put here right ontop of the function. The decorator should go right above the function.